tools/check/go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/pingcap/tidb/_tools
+module github.com/pingcap/tidb/tools/check
```

Comment:
I think maybe it's better to keep it a private moudle?

Reply:
Maybe move it into an internal package, instead of just prefix `_`, @shmsr, what do you think?

Reply:
Sounds good to me.

Reply:
What do you prefer?

> github.com/pingcap/tidb/**internal**/tools/check

 or 

> github.com/pingcap/tidb/tools/**internal**/check

The latter seems to be a better choice.

Reply:
I found that it can be imported anyway. There is no package under this folder. Go will complain about that.

Reply:
I don't think it can solve the problem by putting it in the internal dir, because the internal means only used by our project or moudle. But this module should not be used anywhere.

Reply:
As long as the directory structure does not match the declared path, it does.

Reply:
Ah, okay. But the `idc-jenkins-ci-tidb/unit-test` is failing constantly. I really don't know why it's failing.

Reply:
CI is unstable now. Maybe it is not related to your PR.

Reply:
CI is unstable now. Maybe it is not related to your PR.

Reply:
Okay. Let me change it back to `module github.com/pingcap/tidb/_tools`? or just `module tools`?

Should be fine.

Reply:
OK. Let us just revert it back.

Reply:
Done!